packages/react-email-templates/package.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "emails",
+	"version": "0.0.18",
+	"private": true,
+	"workspaces": [
+		".react-email"
+	],
+	"scripts": {
+		"build": "tsc --build ./",
```

Comment:
for consistency with the `render` lambda, could we use `tsup` here too?
also do the dependencies get zipped correctly? i found that deps get hoisted up to the top level node_modules unless you add the following
```
	"installConfig": {
		"hoistingLimits": "dependencies"
	},
```

Reply:
Earlier I was testing by just building within the `react-email-templates` folder, added this config!